crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1982,6 +1994,17 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PaymentIntentRequest {
 
         let meta_data = get_transaction_metadata(item.request.metadata.clone(), order_id);
 
+        // We pass browser_info only when payment_data exists.
+        // Hence, we're pass Null during recurring payments as payment_method_data[type] is not passed
+        let mut browser_info = None;
```

Comment:
let's avoid mutable variables, you can make use of if else, or if possible methods available on bools